DXMainClient/DXGUI/Multiplayer/GameLobby/GameLobbyBase.cs

```diff
@@ -381,6 +387,63 @@ public override void Initialize()
             AddChild(btnPickRandomMap);
         }
 
+        private void InitializeGameOptionsPanel()
+        {
+            GameOptionsPanel = new XNAPanel(WindowManager);
+            GameOptionsPanel.Name = "GameOptionsPanel";
+            GameOptionsPanel.ClientRectangle = new Rectangle(Width - 411, 12, 399, 289);
+            GameOptionsPanel.BackgroundTexture = AssetLoader.CreateTexture(new Color(0, 0, 0, 192), 1, 1);
+            GameOptionsPanel.PanelBackgroundDrawMode = PanelBackgroundImageDrawMode.STRETCHED;
+
+            loadSaveGameOptionsWindow = new LoadSaveGameOptionsWindow(WindowManager);
```

Comment:
Assign a name for this control to expose it to the INI theming system. Same for `loadSaveGameOptionsMenu` and `btnSaveLoadGameOptions`. (Btw, it'd be good to also rename these if the window is renamed)

Reply:
Done.